synthesis/performance_power_area_result.proto

```diff
@@ -0,0 +1,95 @@
+syntax = "proto2";
+
+package bazel_rules_hdl.ppa;
+
+message PerformancePowerAreaResultProto {
+  // ====== Power: Total ======
+  // The unit scale such as micro (u), nano (n), etc.
+  optional string power_magnitude = 5;
+  // The internal power simulated
+  optional float total_internal_package_power_watts = 1;
+  // The switching power simulated
+  optional float total_switching_package_power_watts = 2;
+  // The leakage power simulated
+  optional float total_leakage_package_power_watts = 3;
+  // The total power simulated
+  optional float total_total_package_power_watts = 4;
```

Comment:
maybe make this a message itself?

message x {
  optional string power_magnitude = 5;
  optional float internal_package_power_watts = 1;
  optional float witching_package_power_watts = 2;
  optional float leakage_package_power_watts = 3;
  optional float total_package_power_watts = 4;
}

message PerformancePowerAreaResultProto {
  optional x total;
  optional x sequential;
  ...
}

Reply:
@grotival Any thoughts here?